app/src/main/java/com/p2p/presentation/tuttifrutti/finalscore/FinalScoreTuttiFruttiFragment.kt

```diff
@@ -0,0 +1,58 @@
+package com.p2p.presentation.tuttifrutti.finalscore
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.fragment.app.activityViewModels
+import androidx.fragment.app.viewModels
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.p2p.R
+import com.p2p.databinding.FragmentTuttiFruttiFinalScoreBinding
+import com.p2p.presentation.basegame.BaseGameFragment
+import com.p2p.presentation.tuttifrutti.TuttiFruttiViewModel
+
+class FinalScoreTuttiFruttiFragment : BaseGameFragment<
+        FragmentTuttiFruttiFinalScoreBinding,
+        TuttiFruttiFinalScoreEvent,
+        FinalScoreTuttiFruttiViewModel,
+        TuttiFruttiViewModel>() {
+    override val gameViewModel: TuttiFruttiViewModel by activityViewModels()
+
+    override val gameInflater: (LayoutInflater, ViewGroup?, Boolean) -> FragmentTuttiFruttiFinalScoreBinding =
+        FragmentTuttiFruttiFinalScoreBinding::inflate
+
+    override val viewModel: FinalScoreTuttiFruttiViewModel by viewModels()
+
+    private lateinit var tuttiFruttiFinalScoreAdapter: TuttiFruttiFinalScoreAdapter
+
+    override fun initUI() {
+        super.initUI()
+        setupObservers()
+        setupScoreRecycler()
+        gameBinding.exitButton.setOnClickListener { viewModel.exit() }
+    }
+
+    override fun setupObservers() = with(gameViewModel) {
+        super.setupObservers()
+        finishedRoundsPointsInfos.observe(viewLifecycleOwner) {
```

Comment:
This is the list set in the TuttiFruttiViewModel right?
Every item of this list is the result for one player in one individual round. What does this mean? That we have to summarize all the points for all the user's rounds.

So if we have two players, and they play two rounds, we could finish with something like this in `finishedRoundsPointsInfos`:

```kotlin
[{ player: 'pepe', totalPoints: 45 }, { player: 'lola', totalPoints: 35 }, { player: 'pepe', totalPoints: 15}, { player: 'lola', totalPoints: 40 } ]
```

Reply:
JAJAJAJAJ LO DIJE EN INGLES PERDON la costumbre

Reply:
JAJAJAJAJ no puede ser

Reply:
JAJAJA entrega los dolares vale (?